Tests/FeedStoreIntegrationTests.swift

```diff
@@ -27,60 +28,66 @@ class FeedStoreIntegrationTests: XCTestCase {
 	}
 	
 	func test_retrieve_deliversEmptyOnEmptyCache() throws {
-//		let sut = try makeSUT()
-//
-//		expect(sut, toRetrieve: .empty)
+		let sut = try makeSUT()
+		
+		expect(sut, toRetrieve: .empty)
 	}
 	
 	func test_retrieve_deliversFeedInsertedOnAnotherInstance() throws {
-//		let storeToInsert = try makeSUT()
-//		let storeToLoad = try makeSUT()
-//		let feed = uniqueImageFeed()
-//		let timestamp = Date()
-//
-//		insert((feed, timestamp), to: storeToInsert)
-//
-//		expect(storeToLoad, toRetrieve: .found(feed: feed, timestamp: timestamp))
+		let storeToInsert = try makeSUT()
+		let storeToLoad = try makeSUT()
+		let feed = uniqueImageFeed()
+		let timestamp = Date()
+		
+		insert((feed, timestamp), to: storeToInsert)
+		
+		expect(storeToLoad, toRetrieve: .found(feed: feed, timestamp: timestamp))
 	}
 	
 	func test_insert_overridesFeedInsertedOnAnotherInstance() throws {
-//		let storeToInsert = try makeSUT()
-//		let storeToOverride = try makeSUT()
-//		let storeToLoad = try makeSUT()
-//
-//		insert((uniqueImageFeed(), Date()), to: storeToInsert)
-//
-//		let latestFeed = uniqueImageFeed()
-//		let latestTimestamp = Date()
-//		insert((latestFeed, latestTimestamp), to: storeToOverride)
-//
-//		expect(storeToLoad, toRetrieve: .found(feed: latestFeed, timestamp: latestTimestamp))
+		let storeToInsert = try makeSUT()
+		let storeToOverride = try makeSUT()
+		let storeToLoad = try makeSUT()
+		
+		insert((uniqueImageFeed(), Date()), to: storeToInsert)
+		
+		let latestFeed = uniqueImageFeed()
+		let latestTimestamp = Date()
+		insert((latestFeed, latestTimestamp), to: storeToOverride)
+		
+		expect(storeToLoad, toRetrieve: .found(feed: latestFeed, timestamp: latestTimestamp))
 	}
 	
 	func test_delete_deletesFeedInsertedOnAnotherInstance() throws {
-//		let storeToInsert = try makeSUT()
-//		let storeToDelete = try makeSUT()
-//		let storeToLoad = try makeSUT()
-//
-//		insert((uniqueImageFeed(), Date()), to: storeToInsert)
-//
-//		deleteCache(from: storeToDelete)
-//
-//		expect(storeToLoad, toRetrieve: .empty)
+		let storeToInsert = try makeSUT()
+		let storeToDelete = try makeSUT()
+		let storeToLoad = try makeSUT()
+		
+		insert((uniqueImageFeed(), Date()), to: storeToInsert)
+		
+		deleteCache(from: storeToDelete)
+		
+		expect(storeToLoad, toRetrieve: .empty)
 	}
 	
 	// - MARK: Helpers
-	
-	private func makeSUT() throws -> FeedStore {
-		fatalError("Must be implemented")
+	private func makeSUT(file: StaticString = #file, line: UInt = #line) throws -> FeedStore {
+		let storeBundle = Bundle(for: CoreDataFeedStore.self)
+		let sut = try CoreDataFeedStore(bundle: storeBundle)
```

Comment:
Missing memory leak tracking:

```suggestion
		let sut = try CoreDataFeedStore(bundle: storeBundle)
		trackForMemoryLeaks(sut, file: file, line: line)
```

Reply:
Added! :)